### DISCUSSION TITLE
UAC disabled by default?

### DISCUSSION THREAD

**Discussion by martin-braun** [utmapp/UTM#5323] [Windows]
I'm not sure this is the case, but I installed the official Windows 11 image and UAC was disabled by default, which is quite shocking. If this is not an issue on my configuration, I wonder why UTM disables UAC on a fresh Windows installation and I would advice against that for security reasons.

**Comment by mattieb** [comment]
I've seen this too, using ISO builds I've made with [my own scripts](https://github.com/utmapp/UTM/discussions/5324). If you start a Windows Terminal session, it'll be an Administrator session.

If I use the same ISO in VMware Fusion Player, I don't have the same problem.

UTM test installation:

<img width="1092" alt="image" src="https://github.com/utmapp/UTM/assets/56017/4609a000-6f21-4f3c-835c-2fe0b65b7c12">

VMware Fusion Player test installation (interestingly, the UTM VM didn't ask me to connect to a network during setup; this one did, so I used [the BypassNRO method](https://answers.microsoft.com/en-us/insider/forum/all/set-up-windows-11-without-internet-oobebypassnro/4fc44554-b416-4ecb-8961-6f79fd55ae0f) to get through and just set up a local account for testing):

<img width="1136" alt="image" src="https://github.com/utmapp/UTM/assets/56017/fb3598c2-2258-4f75-b4e9-da7071ade63d">

**Comment by mattieb** [comment]
If you want to change it, the setting is in "User Account Control Settings" (search in the Start menu). You want it at "Notify me only when apps try to make changes to my computer (default)", which is the default I'm used to:

<img width="1136" alt="image" src="https://github.com/utmapp/UTM/assets/56017/7c8ff692-7123-4c04-9a49-1b66e8c80673">

**Reply by martin-braun** [reply]
I know how to change it and did it eventually, but I still think this might be an issue that should be fixed. Maybe there is a legit reason to have UAC disabled by default, but I can't make one up.

**Comment by sertorsol** [comment]
still the same
